scripts/helpers/multiedit.py

```diff
@@ -25,38 +25,54 @@
 
 else:
     #
-    # Now build up arguments
+    # Determine the changes the user has requested. Convert each change to
+    # its backend format.
     #
 
     fields = args['fields'].split(',')
     changes = {}
 
+    is_get_changes_success = True
     for f in fields:
-        # The value should be really filterEdited. This logic is missing and
-        # should be added to fix changing the inability to save Component
-        # on the multiedit page.
-        changes[allFields[f].savename] = args[f]
+        if not is_get_changes_success:
```

Comment:
You could just "break" from the loop when setting this condition instead of checking it continuously